cmd/cloud/organization.go

```diff
@@ -327,6 +330,41 @@ func teamUpdate(cmd *cobra.Command, out io.Writer, args []string) error {
 	return team.UpdateTeam(id, teamName, teamDescription, out, astroCoreClient)
 }
 
+func newOrganizationTeamUpdateRoleCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-org-role [team-id]",
```

Comment:
I'm open to other cmd names lmk what you think

Reply:
I think it should be part of the team update command because the org role is sort of property for the org team resource, and also it is a flag for the team create command, so it should also be the same flag for the team update command, wdyt?

Reply:
yeah makes sense done